scripts/embed.lua

```diff
@@ -136,6 +147,16 @@
 		end
 	end
 
+-- Write he user modules.
+
+	if #userModuleFiles > 0 then
+		local userModules = {}
+		for _, userModuleFile in ipairs(userModuleFiles) do
+			userModules = table.join(userModules, dofile(userModuleFile))
+		end
+		appendScript(result, "return {" .. table.implode(userModules, "\\\"", "\\\"", ",\\n") .. "}")
```

Comment:
Confused by this line. Why shouldn't `_user_modules.lua` look exactly like `_modules.lua` (already a table, also like `_manifest.lua` and `_tests.lua`) and be loaded the same way? Not a big deal but would prefer the consistency?